packages/react/src/components/Backdrop/__tests__/Backdrop.test.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Let's remove the phrase All Rights Reserved in the new files as well.

```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
As I mentioned earlier https://github.com/wso2/oxygen-ui/pull/134#discussion_r1229243882, this will be fixed in a chore PR